setup.py

```diff
@@ -68,6 +68,7 @@ def rglob(path, glob):
         "sympy",
         "tinydb",
         "rbfopt",
+        "scipy",
```

Comment:
One minor change, you should move "scipy" up in the list to keep the requirements in alphabetical order.